storage/client.go

```diff
@@ -334,6 +361,12 @@ func IsValidStorageAccount(account string) bool {
 	return validStorageAccount.MatchString(account)
 }
 
+// IsValidCosmosAccount checks if the Cosmos account name is valid.
+// See https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-manage-database-account
+func IsValidCosmosAccount(account string) bool {
```

Comment:
Does this actually need to be exported, i.e. do we expect customers to call this?  If not can we fold the contents into `NewCosmosClient()`.

Reply:
I do see we have something similar for storage `IsValidStorageAccount()`, so it's fine to keep this for sake of parity.